workshop/bx_login.sh

```diff
@@ -8,7 +8,7 @@ if [ -z $CF_SPACE ]; then
 fi
 
 
-if [ -z "$BLUEMIX_API_KEY" || [ -z "$BLUEMIX_NAMESPACE" ]; then
+if [ -z "$BLUEMIX_API_KEY" ] || [ -z "$BLUEMIX_NAMESPACE" ]; then
```

Comment:
Don't forget to set the environment variables here https://travis-ci.org/IBM/kube101/settings